app/models/sorn.rb

```diff
@@ -68,6 +70,8 @@ class Sorn < ApplicationRecord
 
   def get_xml
     if xml_url.present? and xml.blank?
+      sleep 1
```

Comment:
I can't recall if this was the case, but didn't you mention there was a potential optimization about where this sleep was?

Reply:
I mentioned that moving it here would be a good idea!

Reply:
Great!

Reply:
I also figured out how to stub it in the tests, so they run faster.